src/test/java/io/jenkins/plugins/artifact_manager_s3/S3BlobStoreConfigTests.java

```diff
@@ -0,0 +1,90 @@
+package io.jenkins.plugins.artifact_manager_s3;
+
+import java.util.logging.Logger;
+import io.jenkins.plugins.artifact_manager_jclouds.JCloudsArtifactManagerFactory;
+import org.apache.commons.lang.StringUtils;
+import org.junit.After;
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.JenkinsRule;
+import jenkins.model.ArtifactManagerConfiguration;
+import static io.jenkins.plugins.artifact_manager_s3.S3BlobStore.KEY_CONTAINER;
+import static io.jenkins.plugins.artifact_manager_s3.S3BlobStore.KEY_PREFIX;
+import static io.jenkins.plugins.artifact_manager_s3.S3BlobStore.KEY_REGION;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class S3BlobStoreConfigTests {
```

Comment:
Can just make the code simple enough that this whole test suite is unnecessary and can be deleted.